tuf/api/metadata.py

```diff
@@ -87,7 +87,7 @@ class Metadata(Generic[T]):
     Provides methods to convert to and from dictionary, read and write to and
     from file and to create and verify metadata signatures.
 
-    Metadata[T] is a generic container type where T can be any one type of
+    ``Metadata[T]`` is a generic container type where T can be any one type of
     [Root, Timestamp, Snapshot, Targets]. The purpose of this is to allow
```

Comment:
I suppose you could double-backtick all of these as well to show that they are actual python types/classes?